CHANGELOG-0.6.md

```diff
@@ -7,6 +7,7 @@
 - [#1154](https://github.com/epiphany-platform/epiphany/issues/1154) - Node exporter is not installed on logging vms
 - [#1135](https://github.com/epiphany-platform/epiphany/issues/1135) - 2ndquadrant yum repos remain enabled on repository host after teardown
 - [#1169](https://github.com/epiphany-platform/epiphany/issues/1169) - Task 'Get token from master' fails on-prem when calico plugin is used
+- [#1181](https://github.com/epiphany-platform/epiphany/issues/1181) - Configure Ignite to use fixed ports
```

Comment:
This shouldn't go to changelog for 0.7 and after backmerge to 0.6?

Reply:
Added changelog for v0.7 and moved the entry there